meshery

```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
 unzip ${PWD}/meshery.zip 
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
-mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl
+if mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
+  echo "mesheryctl installed"
```

Comment:
@anandrkskd we should update their version of mesheryctl by overwriting their existing binary rather than aborting.

Reply:
Given that this PR has merged, let's open a new issue to track the need to change this behavior.